### DISCUSSION TITLE
UI for 3D movement in editor

### DISCUSSION THREAD

**Discussion by RikKargones** [godotengine/godot-proposals#5156] [3D]
As mentioned in [this](https://github.com/godotengine/godot/issues/53?ysclid=l6rnjsgdw3546275422?ysclid=l6rnjsgdw3546275422) issue, new Godot users may have troubles with navigating in 3D editor, because it's not so intuitive and in some cases needs to open editor settings and change one specific option to work properly. I think UI controls similar from Cinema 4D, which you can press&hold LMB and move mouse to navigate (screenshot below) will be useful to implement in engine. Maybe even adding a tooltip to them, to be super user-frendly here.

![UI](https://user-images.githubusercontent.com/58859344/184477911-206c25a9-736a-497d-9b52-93feb848e0e9.png)

This will help:
- People, who uses trackpads on laptops.
- People, who unfamiliar with Blender or other 3D-modeling programs.
- Give a bit less frustration upon exploring Godot. If user want, he can find instructions for hotkeys in Q&A or Godot issues (Hey, maybe add some brif explanation in docs too? I didn't see what where...).

**Comment by Calinou** [comment]
Personally, I would just display a list of movement keys when starting the 3D editor until you perform any kind of movement.
